src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,56 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * This class encapsulates a command to find a Task in a TaskList.
+ * Extends Command class.
+ */
+public class FindCommand extends Command {
+
+    private String[] searchInput;
```

Comment:
Perhaps can rename the variable to reflect that it's an array, maybe searchKeywords or sth like that?